lib/cdo/shared_constants/pd/shared_workshop_constants.rb

```diff
@@ -94,6 +95,28 @@ module SharedWorkshopConstants
       SUBJECT_VIRTUAL_KICKOFF
     ].freeze
 
+    # Used in create/edit workshop UI
+    MUST_SUPPRESS_EMAIL_SUBJECTS = [
+      SUBJECT_VIRTUAL_KICKOFF,
+      SUBJECT_WORKSHOP_1,
+      SUBJECT_WORKSHOP_2,
+      SUBJECT_WORKSHOP_3,
+      SUBJECT_WORKSHOP_4,
+      SUBJECT_WORKSHOP_1_2,
+      SUBJECT_WORKSHOP_3_4
+    ].freeze
+
+    # Used to suppress post-workshop emails
+    ACADEMIC_YEAR_WORKSHOP_SUBJECTS = [
```

Comment:
could we combine these two lists somehow? Is the reason for both that we think they may divert? For now could we set `MUST_SUPPRESS_EMAIL_SUBJECTS` to `ACADEMIC_YEAR_WORKSHOP_SUBJECTS`?

Reply:
Yeah, in theory they could be different -- I'll set `MUST_SUPPRESS_EMAIL_SUBJECTS` to `ACADEMIC_YEAR_WORKSHOP_SUBJECTS`, good call.